kidviz/views.py

```diff
@@ -1005,9 +1005,15 @@ class WorkQueue(LoginRequiredMixin, ListView):
     paginate_by = 10
 
     def get_queryset(self):
-        return Observation.objects \
+        not_draft_observations = Observation.objects \
             .prefetch_related('students') \
-            .filter(owner=self.request.user, constructs=None)
+            .filter(owner=self.request.user, constructs=None, is_draft=False)
+
+        draft_observations = Observation.objects \
+            .prefetch_related('students') \
+            .filter(owner=self.request.user, is_draft=True)
+
+        return not_draft_observations.union(draft_observations).order_by('id')
```

Comment:
You can perform this query with Q (https://docs.djangoproject.com/en/2.2/topics/db/queries/#complex-lookups-with-q-objects)